server/apps/users/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+class UserProfile(models.Model):
+    """
+    Holds profile information about a user
+    """
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    profile_submitted = models.BooleanField(default=False)
+    age_bracket = models.TextField(default="unspecified")
+    gender = models.TextField(default="unspecified")
+    self_identification = models.TextField(default="unspecified")
+    description = models.TextField(default="")
+    location = models.TextField(default="")
+    comms_review = models.BooleanField(default=False)
```

Comment:
Probably something to discuss with the community what the default on this should be. I'm worried that people will never get feedback if the default is not receiving and people might click away the form but of course opting-in is good practice.

Reply:
Understood. I've left it as it is for now, but ready to change it after discussion. If you'd prefer us to present it with the default set to True, let me know and I can switch it.

Reply:
Let's leave it like this for now and we ask for feedback on it during the community call? :)